README.md

```diff
@@ -210,10 +207,10 @@ functions returns a closure that can be passed to `map` and `mapBy`.
 For example:
 
 ```php
+use \Zicht\Itertools\util\Mappings;
```

Comment:
The use statements in these examples should not contain a prefixed backslash.

PSR-12:
> Import statements MUST never begin with a leading backslash

Reply:
Fixed in 3 locations